code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -3040,6 +3040,13 @@
 	required_reagents = list(CARBON = 1, SACID = 2)
 	required_temp = T0C + 450
 	result_amount = 1
+	
+/datum/chemical_reaction/albuterol
+	name = "Albuterol"
+	id = ALBUTEROL
+	result = ALBUTEROL
+	required_reagents = list(HYPERZINE = 1, INAPROVALINE = 1)
```

Comment:
I would see Dexalin as something more fitting.

Reply:
The only reason I used those two is because dexalin is plasma-based and I figured I would make it a bit simpler to not have another life-saving plasma required chem.